jwt.go

```diff
@@ -36,7 +36,7 @@ type Config struct {
 type AtlassianClaims struct {
 	QSH string `json:"qsh"`
 
-	jwt.StandardClaims
+	jwt.RegisteredClaims
```

Comment:
Heads up that this is a breaking change.

Reply:
Is there a way to update the version or start versioning it?

Reply:
Are you referring to the version of github.com/rbriski/atlassian-jwt? That can be done using a normal git tag.